worker/src/RTC/TcpServer.cpp

```diff
@@ -26,7 +35,10 @@ namespace RTC
 	{
 		MS_TRACE();
 
-		RTC::PortManager::UnbindTcp(this->localIp, this->localPort);
+		if (!fixedPort)
+		{
+			RTC::PortManager::UnbindTcp(this->localIp, this->localPort);
+		}
```

Comment:
@nazar-pc, here and in `UdpSocket.cpp` I don't see where the given fixed `port` is unbinded once socket is closed. Am I missing something?

Reply:
Forget it, such a port is unbinded by the UdpSocketHandler and so on. It's ok.